modules/account-lib/package.json

```diff
@@ -40,6 +40,7 @@
     "@bitgo/sdk-coin-eth2": "^1.2.18",
     "@bitgo/sdk-coin-hbar": "^1.3.18",
     "@bitgo/sdk-coin-near": "^1.3.18",
+    "@bitgo/sdk-coin-osmo": "^1.0.0",
```

Comment:
Do we need this for other services using account-lib? If not, let's remove this.

Reply:
If needed, also needs tsconfig update.